### DISCUSSION TITLE
Remote snoozing does not work

### DISCUSSION THREAD

**Discussion by bchris21** [NightscoutFoundation/xDrip#2592] [Q&A]
Hello everyone,

I would like to start using remote snoozing on my son's xDrip phone.

Even though Feature it is enabled, I try to snooze an active alarm from my phone) follower) but it doesn't work.

Both phones are on same Wi-Fi.

Am I missing something?

Thanks,

Christos

**Comment by tzachi-dar** [comment]
Do you have location enabled on both phones

**Reply by bchris21** [reply]
Is it mandatory to have it enabled constantly? Or just allow the permissions for xDrip (already allowed on both master and follower).

![Screenshot_20230101_213531_com.google.android.permissioncontroller.jpg](https://user-images.githubusercontent.com/4725280/210183927-fe3106aa-d781-45af-85fa-7c62a628db1e.jpg)

**Reply by Navid200** [reply]
Would you please enable the location service on both phones constantly and report back if it still does not work?

**Comment by Navid200** [comment]
Would you please post a screenshot of the following page on your (follower) phone?

Settings &#8722;> Hardware Data Source

**Reply by bchris21** [reply]
Hello, please find screenshots requested below.

![Screenshot_20230101_213117_com.eveningoutpost.dexdrip.jpg](https://user-images.githubusercontent.com/4725280/210183865-281a00fb-624d-4364-a09f-8a99159741e0.jpg)

![Screenshot_20230101_213206.jpg](https://user-images.githubusercontent.com/4725280/210183868-b3cd81ac-721c-411e-a4ae-07ae57ad5146.jpg)
